### DISCUSSION TITLE
hi there if i have an established table [Nov 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4382] [Discord Threads]
> 🙋‍♂️ **unkn**

**Comment by o1lab** [comment]
> 🙋‍♂️ **mank**  
#### also, is it possible to hide certain columns from certain roles?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1036407874785648700)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
